patch-setup-config.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
OK, this works as a temporary hack, but the general solution shouldn't rely on shell or perl, and we also can't rely on hard-coded values like `2.0.0.0`.